styles.css

```diff
@@ -0,0 +1,594 @@
+  color: white;
+}
+
+#navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px;
+  height: 75px;
+}
+
+#navbar #hamburger {
+  width: 25px;
+  cursor: pointer;
+  transition: transform 0.3s ease-in-out;
+}
+
+#hamburger-icon {
+  width: 36px;
+}
+
+#navbar #hamburger-icon.close {
+  display: none;
+}
+
+#navbar #cancel-icon {
+  position: absolute;
+  display: none;
+  z-index: 9999;
+  width: 21px;
+}
+
+#navbar #cancel-icon.show {
+  display: block;
+}
+
+.nav-menu {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.8);
+  padding: 100px 0;
+  text-align: center;
+  transition: left 0.3s ease-in-out;
+  backdrop-filter: blur(10px);
+}
+
+.nav-menu li a {
+  color: #ec5242;
+  padding: 8px;
+}
+
+.nav-menu li #campaign {
+  border: 2px solid #ec5242;
+}
+
+.nav-menu.show {
+  left: 0;
+}
+
+#hero {
+  background-color: rgb(247, 247, 248);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  font-weight: 400;
+}
+
+#hero-content {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  padding-top: 50px;
+  gap: 32px;
+  color: #4d4c4c;
+}
+
+.hero-about h1 {
+  text-align: center;
+  align-items: center;
+}
+
+#hero h1 {
+  color: #ec5242;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+#hero h1 .main {
+  background-image: url(images/h1-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-weight: 800;
+  line-height: 150%;
+  width: 90%;
+  font-size: 32px;
+}
+
+#hero .info {
+  padding: 1em 0.5em;
+  line-height: 150%;
+  font-size: 13px;
+  width: 85%;
+  height: auto;
+  text-align: justify;
+  border: 4px solid rgb(255, 255, 255);
+}
+
+#hero h2 {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+#hero #hero-time {
+  font-size: 29px;
+  font-weight: 800;
+  width: 100%;
+  margin-bottom: 4px;
+}
+
+#program {
+  background-image: url("images/program-bg-pattern.png");
+  background-repeat: repeat;
+  background-color: rgb(39, 42, 50);
+  height: auto;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  padding-top: 10%;
+}
+
+#program h3 {
+  font-weight: bold;
+}
+
+#program .programs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  width: 90%;
+  margin-bottom: 25px;
+}
+
+#program .program-item {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background: rgba(255, 255, 255, 0.1);
+  height: auto;
+  gap: 20px;
+  padding: 12px;
+}
+
+#program .program-icon {
+  font-size: 32px;
+  width: 20%;
+  font-weight: 700;
+  color: #f00;
+}
+
+#program .program-name {
+  font-size: 18px;
+  width: 30%;
+  color: #f3200d;
+}
```

Comment:
- [ ] Please make sure to use the colors provided by the design guideline. Here the color should not be `#f3200d` and `#f00` but `#ec5242`. 

**Provided colors**
![image](https://user-images.githubusercontent.com/74411135/235320967-d9d5a6ac-07fa-4927-a7c2-cd6e0dbb662a.png)

That will make your project look way better as the Figma design is done by a professional designer 😮 !